tweb/src/main/frontend/src/scss/fundrequest/_alert.scss

```diff
@@ -92,6 +92,11 @@
     }
 }
 
+.alert-body {
+  white-space: nowrap;
```

Comment:
Nowrap is not an option. Because the content of can be wider, and the screensize can be smaller.
<img width="494" alt="screen shot 2018-06-18 at 13 31 23" src="https://user-images.githubusercontent.com/1853096/41533940-e6c81c4e-72fc-11e8-9d4f-dae3a6c7a0d3.png">

Reply:
Makes sense. 

But that makes this bug-report a bit ambigous.

I understand that it *should* wrap in some situations, but not in others. Could you specify the request a bit better, in that case?

1. Expectation when the message is `"ETH address saved!"`
1. Expectation is expected when the message is `_.repeat("ETH address saved!", 80, " ")`
1. Expectation is expected when the message is `_.repeat("unbreakable", 80)`

One could also argue that a (documented?) requirement for the texts that are added to alerts
are not longer than X characters and if they are, they should be manually broken with `<br/>` by the caller. :woman_shrugging:

Reply:
The bug report also states that 'on desktop' it should not span multiple line. Which implies that not in all cases it should be on 1 line all the time. 
The text should be on 1 line when possible, when it doesn't fit on the screen it makes sense that there is a way to view all the text. We also didn't mention that the closing icon should not overlap the text, but this is also common sense.
If the bug isn't clear it's always possible to ask questions. It's key to think along with the project and find the best solutions regarding usability.

Reply:
Using nowrap is not the solution. The alert shows text on multiple line because of the position of the closing button, doesn't matter how long the text is. If you remove the closing button, the alert behaves like it should. A possible solution is to add extra right margin to the alert, and position the closing button absolute.

Reply:
@Simsteve7 I've implemented your suggestion at #440. It is a new PR because I renamed the branch so that travis can process the PR.